### PR TITLE
Classification fixes

### PR DESCRIPTION
**Pull request by psinger**
This PR:

- adds log loss metric for classification
- fixes errors after previous deepspeed merge

### CODE REVIEW

File: llm_studio/src/metrics/text_causal_classification_modeling_metrics.py [hunk 1]
```diff
@@ -47,6 +62,7 @@ class Metrics:
     _metrics = {
         "AUC": (auc_score, "max", "mean"),
         "Accuracy": (accuracy_score, "max", "mean"),
+        "LogLoss": (logloss_score, "max", "mean"),
```
Comment (maxjeblick) [inline llm_studio/src/metrics/text_causal_classification_modeling_metrics.py:65]: should be `"min"` ?

**Review by maxjeblick — APPROVED** [review]
Thanks for the fixes, LGTM (I've left one comment).

Also, could you have a look at this line, should probably read `>0.5` ?
`preds = np.array((preds > 0.0)).astype(int).astype(str).reshape(-1)`

**Comment by psinger** [comment]
> Thanks for the fixes, LGTM (I've left one comment).
> 
> Also, could you have a look at this line, should probably read `>0.5` ? `preds = np.array((preds > 0.0)).astype(int).astype(str).reshape(-1)`

it is logits, so I think `>0.0` should be correct

**Comment by maxjeblick** [comment]
> it is logits, so I think >0.0 should be correct

True, it is fine then. Thanks for the clarification.
